common/lib/xmodule/xmodule/js/spec/video/video_bumper_spec.js

```diff
@@ -36,7 +36,7 @@
         });
 
         it('can show the main video on error', function (done) {
-            state.el.trigger('error');
+            state.el.triggerHandler('error');
```

Comment:
Karma has a global event handler for events named "error" which expects the event data to be a string and so fails when this event is triggered. So we use triggerHandler which only invokes handlers attached via jQuery.